third_party/optimized-svg-icons/README.amp

```diff
@@ -1,17 +0,0 @@
-URL: http://codepen.io/anon/pen/KVZNGp
```

Comment:
This seems premature? Isn't 0.1 still using these resources?

Reply:
I agree, thanks for the reminder!  I will revert this part!